parser/peaklistReader/PeakListWrapper.py

```
@@ -131,8 +142,10 @@ def unzip_peak_lists(zip_file, out_path='.'):
         :return: path to resulting folder
         """
         if zip_file.endswith(".zip"):
-            zip_ref = zipfile.ZipFile(zip_file, 'r')
-            unzip_path = os.path.join(str(out_path), ntpath.basename(zip_file + '_unzip'))
+            zip_ref = zipfile.ZipFile(zip_file, "r")
+            unzip_path = os.path.join(
+                str(out_path), ntpath.basename(zip_file + "_unzip")
+            )
             zip_ref.extractall(unzip_path)
```

Ruff (0.13.1)

Uses of `tarfile.extractall()`

(S202)